src/controls/TimelineControl.java

```diff
@@ -3,92 +3,110 @@
 	 * Checks if while creating a timeline a name has been chosen.
-	 * @param name Name of timeline
-	 * @return true if name is not a null
+	 *
+	 * @param name
+	 *            Name of timeline
+	 * @return true if name length is not 0
 	 */
 	private boolean isNameCorrect(String name) {
 		if (name.length() == 0) {
-			System.out.println("Please chose a name for your timeline");
+			Alert alert = new Alert(AlertType.ERROR);
+			alert.setTitle("Error in timeline name");
+			alert.setHeaderText("Please choose a name for your timeline");
+			alert.show();
 			return false;
-		}
-		else {
+		} else {
 			return true;
 		}
 	}
-	
+
 	/**
 	 * Checks if while creating a timeline a start date has been chosen.
-	 * @param start Date of when timeline starts
-	 * @return true if start date is not a null
+	 *
+	 * @param start
+	 *            Date of when timeline starts
+	 * @return true if start date is not temp date
 	 */
 	private boolean isStartCorrect(LocalDateTime start) {
-		if (start == null) {
-			System.out.println("Please chose a start date for your timeline");
+		LocalDateTime temp = LocalDateTime.parse("0000-01-01T03:00:01");
+		if (start.equals(temp)) {
+			Alert alert = new Alert(AlertType.ERROR);
+			alert.setTitle("Error in timeline start date");
+			alert.setHeaderText("Please choose a start date for your timeline");
+			alert.setContentText("Start year must be 4 numbers long. (Ex. 0001,0002...2016,2017)");
+			alert.show();
 			return false;
-		}
-		else {
+		} else {
 			return true;
 		}
 	}
-	
+
 	/**
 	 * Checks if while creating a timeline an end date has been chosen.
-	 * @param end Date of when timeline ends
-	 * @return true if end date is not a null
+	 *
+	 * @param end
+	 *            Date of when timeline ends
+	 * @return true if end date is not temp date
 	 */
-	private boolean isEndCorrect (LocalDateTime end) {
-		if (end == null) {
-			System.out.println("Please chose an end date for your timeline");
+	private boolean isEndCorrect(LocalDateTime end) {
+		LocalDateTime temp = LocalDateTime.parse("0000-01-01T03:00:01");
+		if (end.equals(temp)) {
+			Alert alert = new Alert(AlertType.ERROR);
+			alert.setTitle("Error in timeline end date");
+			alert.setHeaderText("Please choose an end date for your timeline.");
+			alert.setContentText("End year must be 4 numbers long. (Ex. 0001,0002...2016,2017)");
+			alert.show();
 			return false;
-		}
-		else {
+		} else {
 			return true;
 		}
 	}
-	
+
 	/**
 	 * Checks if all input made by user is correct
-	 * @param name Name of timeline
-	 * @param start Date of when timeline starts 
-	 * @param end Date of when timeline ends
+	 *
+	 * @param name
+	 *            Name of timeline
+	 * @param start
+	 *            Date of when timeline starts
+	 * @param end
+	 *            Date of when timeline ends
 	 * @return
 	 */
 	private boolean isCorrectInput(String name, LocalDateTime start, LocalDateTime end) {
 		if (isNameCorrect(name) && isStartCorrect(start) && isEndCorrect(end)) {
```

Comment:
IMO this method could simply return `isNameCorrect(name) && isStartCorrect(start) && isEndCorrect(end)`